cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/CassandraCluster.java

```diff
@@ -93,7 +93,8 @@
     );
 
     private static final Map<String, String> executorEnv = unmodifiableHashMap(
-        tuple2("JAVA_OPTS", "-Xms256m -Xmx256m")
+        tuple2("JAVA_OPTS", "-Xms256m -Xmx256m"),
+        tuple2("MESOSPHERE_LD_LIBRARY_PATH", "/opt/mesosphere/lib")
```

Comment:
This value is appended to the path at executor start time. You're right that it's normally not there and will only be there in DCOS.

I want to test this out on DCOS and mesos to make sure that it works in both locations then I'll merge it.